politeiawww/cmd/piwww/editproposal.go

```diff
@@ -27,10 +27,11 @@ type EditProposalCmd struct {
 		Markdown    string   `positional-arg-name:"markdownfile"`          // Proposal MD file
 		Attachments []string `positional-arg-name:"attachmentfiles"`       // Proposal attachments
 	} `positional-args:"true" optional:"true"`
-	Random bool   `long:"random" optional:"true"` // Generate random proposal data
-	Name   string `long:"name" optional:"true"`
-	RFP    bool   `long:"rfp" optional:"true"`    // Insert a LinkBy timestamp to indicate an RFP
-	LinkTo string `long:"linkto" optional:"true"` // Censorship token of prop to link to
+	Random   bool         `long:"random" optional:"true"` // Generate random proposal data
+	Name     string       `long:"name" optional:"true"`
+	RFP      bool         `long:"rfp" optional:"true"`      // Insert a LinkBy timestamp to indicate an RFP
+	LinkTo   string       `long:"linkto" optional:"true"`   // Censorship token of prop to link to
+	Category v1.CategoryT `long:"category" optional:"true"` // Proposal category
```

Comment:
This should be a string and accept both the human readable format or the code.  See the `setproposalstatus` command as an example.

Reply:
Same for newproposal.